lib/galaxy/model/migrations/README.md

```diff
@@ -0,0 +1,70 @@
+
+Since Galaxy uses branch labels to distinguish between the galaxy and the install models, in most cases, you'll need to identify the target branch to which your command should be applied. Use branch labels: `gxy` for the galaxy model, and `tsi` for install model.
+
+### Examples of usage:
+
+Remember to first backup your database(s).
+
+#### To upgrade:
+```
+./migrate_db.sh upgrade gxy@head  # upgrade gxy to head revision
+./migrate_db.sh upgrade gxy@+1  # upgrade gxy to 1 revision above current
+./migrate_db.sh upgrade [revision identifier]  # upgrade branch to a specific revision
+./migrate_db.sh upgrade [revision identifier]+1  # upgrade branch to 1 revision above specific revision
+./migrate_db.sh upgrade heads  # upgrade gxy and tsi to head revisions
+```
+
+#### To downgrade:
+```
+./migrate_db.sh downgrade gxy@base  # downgrade gxy to base (empty db with empty alembic table)
+./migrate_db.sh downgrade gxy@-1  # downgrade gxy to 1 revision below current
+./migrate_db.sh downgrade [revision identifier]  # downgrade branch to a specific revision
+./migrate_db.sh downgrade [revision identifier]-1  # downgrade branch to 1 revision below specific revision
+```
+Check [Alembic documentation](https://alembic.sqlalchemy.org/en/latest/branches.html#working-with-branches) for more examples.
+
+Note: relative upgrades and downgrades without a revision identifier are not supported - i.e., you cannot `upgrade +1` or `downgrade -1` without providing a revision identifier. However, you can upgrade both branches to their latest versions (head revisions) without providing a branch label: `upgrade heads`.
+
+### Upgrading from SQLAlchemy Migrate
+
+Galaxy no longer supports SQLAlchemy Migrate. To upgrade to Alembic, follow these steps:
+
+1. Backup your database(s).
+
+2. Verify that your database is at the latest SQLAlchemy Migrate version. If you have a combined database, it should be version 180 (check the `migrate_version` table). If you have separate galaxy model and install model databases, your galaxy version should be 180, and your install model version should be 17. 
+
+3. If your database is not current, you need to update it first. For that, you'll need to check out a previous version of the code base: use commit 7328ca0da9977bc31cd074464a2e59411a5cdcf8 or later, but prior to [TODO: insert Alembic commit]. Then you can run the old `manage_db.sh` script to bring your database(s) up to date. Once your database has the latest SQLAlchemy Migrate version, you can check out the version of the code base that you had prior to downgrading, and proceed to migrating to Alembic.
```

Comment:
That's too complicated. People should be able to go to the next release and run `./manage_db.sh upgrade`

Reply:
Maybe I phrased it in a confusing way. The idea is (as in my comment below), to use 22.01, you need to be current - i.e., you need to be on 21.09. You cannot "jump" over releases. If you are not - just make sure you are current on 21.09, then upgrade. If this is acceptable, I'll rephrase the readme.

Reply:
Done and done: they can go to the next release and run `./manage_db.sh upgrade` (provided they were up-to-date on the previous release; otherwise they just run the same command on that release). And the steps in README have been updated to be simple and straightforward.

Reply:
Excellent! I'll give the test upgrade another go